src/snowflake/snowpark/mock/mock_analyzer.py

```diff
@@ -170,11 +164,8 @@ def analyze(
         if expr_to_alias is None:
             expr_to_alias = {}
         if isinstance(expr, GroupingSetsExpression):
-            return grouping_set_expression(
-                [
-                    [self.analyze(a, expr_to_alias, parse_local_name) for a in arg]
-                    for arg in expr.args
-                ]
+            raise NotImplementedError(
+                "[Local Testing] GroupingSetsExpression is not implemented."
```

Comment:
`GroupingSetsExpression` etc are internal names, it might be unclear to users how this is related to the dataframe operations they have.